package.json

```diff
@@ -45,7 +45,7 @@
     "debug": "^2.1.1",
     "depd": "^1.0.0",
     "inflection": "^1.6.0",
-    "loopback-connector": "^2.1.0",
+    "loopback-connector": "git+ssh://git@github.com:agriwebb/loopback-connector.git#3618f871569077e8faca3f2daaf3d94246133cab",
```

Comment:
was going to suggest using `git+https` but it seems we use both with no real consistency so 🤷